src/commands/Birthday/birthday.ts

```diff
@@ -120,8 +120,8 @@ export const BirthdayApplicationCommandMentions = {
 	Test: chatInputApplicationCommandMention('birthday', 'test', envParseString('COMMANDS_BIRTHDAY_ID'))
 } as const;
 
-function registerBirthdayCommand(builder: SlashCommandBuilder) {
-	return applyLocalizedBuilder(builder, 'commands/birthday:birthday')
+function registerBirthdayCommand(builder: SlashCommandBuilder, description: string) {
+	return applyDescriptionLocalizedBuilder(builder, description)
```

Comment:
```suggestion
function registerBirthdayCommand(builder: SlashCommandBuilder) {
	return applyDescriptionLocalizedBuilder(builder, this.description)
```

Reply:
since the function is not part of the class, this. is not accessible...?

Reply:
I suggest that you put him in the class 👍🏼